packages/account/src/Components/poi/poi-country-selector/__tests__/poi-country-selector.spec.tsx

```diff
@@ -35,6 +41,7 @@ describe('<CountrySelector/>', () => {
         expect(screen.getByText('In which country was your document issued?')).toBeInTheDocument();
         expect(screen.getByText('Country')).toBeInTheDocument();
 
+        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
```

Comment:
I removed the `container` and instead of that I added `data-testid='dt_auto_complete'` to the `Autocomplete` component in the `packages/account/src/Components/poi/poi-country-selector/poi-country-selector.jsx` file
```
<Autocomplete
    className={classNames({ 'external-dropdown': is_from_external })}
    {...field}
    name='country_input'
    data-testid='dt_auto_complete'
```
then I get the element by testid and fixed the eslint issue!

Reply:
understand will fix it 🙏

Reply:
@niloofar-deriv, @suisin-deriv, I believe these changes have changed the logic of the test case. Hardcoding the `test-id` on the `AutoComplete` component would mean the component will always be in the document. This should at least be dynamically passed in to the component and not added directly to the `AutoComplete` component. Please confirm this.